translations/it/pack/tfa/hote.json

```diff
@@ -2,14 +2,14 @@
     {
         "code": "04192",
         "name": "Intrepido",
-        "text": "Se questa prova di abilità ha successo, aggiungi Intrepido alla tua area di gioco come supporto con il testo:\n\"Ricevi +1 [lore], +1 [strength] e +1 [agility].\n<b>Obbligo</b> - Alla fine del round: Scarta Intrepido.\"",
+        "text": "Se questa prova di abilità ha successo, aggiungi Intrepido alla tua area di gioco come supporto con il testo:\n\"Ricevi +1 [intellect], +1 [combat] e +1 [agility].\n<b>Obbligo</b> - Alla fine del round: Scarta Intrepido.\"",
         "traits": "Innato."
     },
     {
         "code": "04193",
         "flavor": "\"Cosa vuoi che ti benedica... ?\"",
         "name": "Munizioni Modificate",
-        "text": "Veloce. Gioca solo durante il tuo turno.\nAssegna le Munizioni Modificate a un supporto [[Arma da Fuoco]] controllato da un investigatore nel tuo luogo. Colloca 2 munizioni su quel supporto. Limite di 1 per supporto.\nGli attacchi effettuati dal supporto con questo evento infliggono +1 danno attaccando i nemici [[Mostro]].",
+        "text": "Veloce. Gioca solo durante il tuo turno.\nAssegna le Munizioni Modificate a un supporto [[Arma]] [[da]] [[Fuoco]] controllato da un investigatore nel tuo luogo. Colloca 2 munizioni su quel supporto. Limite di 1 per supporto.\nGli attacchi effettuati dal supporto con questo evento infliggono +1 danno attaccando i nemici [[Mostro]].",
```

Comment:
should be all one thing with spaces in the [[ ]]

Reply:
It didn't show properly when it was all one thing earlier.
It was showing on the site as [[text text text] without bold or italics

Reply:
the fix is not live yet but its better to look a bit messy than to fix again properly later.